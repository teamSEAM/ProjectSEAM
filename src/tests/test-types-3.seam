#Float to int and int to float and string join#
entity World:
	int my_int
	float my_float
	string s1

	func start():
		my_float=1.0
		my_int=3
		s1="YO"
		screen.out(convert.float_to_str(convert.int_to_float(my_int)+my_float))
		screen.out(convert.int_to_str(convert.float_to_int(my_float)+my_int))
		screen.out(string_join(s1,s1))
