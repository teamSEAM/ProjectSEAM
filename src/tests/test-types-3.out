4
4
YOYO
